.github/workflows/test.yml

```diff
@@ -0,0 +1,41 @@
+name: Test
+
+on:
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron: '0 22 * * 3'
+  workflow_call:
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    name: Test
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version:
+          - '20.19'
+        os:
+          - macos-latest
+          - ubuntu-22.04
+          - windows-latest
+    runs-on: "${{ matrix.os }}"
+    steps:
+      - name: Install Linux Dependencies
+        if: ${{ matrix.os == 'ubuntu-22.04' }}
+        run: sudo apt-get update && sudo apt-get install --no-install-recommends -y libasound2 libgtk-3-0 libnss3 libxss1 libxtst6 xvfb libgbm-dev
```

Comment:
Do we need these on this repo or is this just a result of copy-pasting a working config?

Reply:
Yeah, I just copied this over. We can probably live without, I'll delete them!